src/ripple/app/ledger/LedgerEntrySet.h

```diff
@@ -338,6 +347,21 @@ class LedgerEntrySet
 
     bool checkState (SLE::pointer state, bool bSenderHigh,
         Account const& sender, STAmount const& before, STAmount const& after);
+
+    STAmount subtractFTCredits (Account const& main,
+                                Account const& other,
+                                STAmount const& amount) const;
+};
+
+class EnableFundingTableGuard
+{
+private:
+    LedgerEntrySet& les_;
+    bool shouldDisable_{false};
```

Comment:
the `bool shouldDisable_` member makes nesting the guard safe. We never nest it (and I can't think of why I would want to). I prefer to keep this, as this error would be difficult to catch if the guard is ever nested and the cost of the bool seems worth it.

For a name I like `enabled_` better than `enable_`. It says if this guard enabled the table or not.